src/app/(protected)/dashboard/page.tsx

```diff
@@ -0,0 +1,52 @@
+"use client";
+
+import ConfirmationModal from "@/components/confirmation-modal";
+import UserCheckboxGroup from "@/components/user-checkbox-group";
+import { contestants } from "@/contestants";
+import { Button } from "@nextui-org/react";
+import { useState } from "react";
+
+const page = () => {
+	const [position, setPosition] = useState(1);
+	const [president, setPresident] = useState("");
```

Comment:
Using that for demo since I don't know how the API is going to send the data and how the response should be handles, but would probably plug it to a form.